### PR TITLE
feat: Introduce PR welcome bot

### PR DESCRIPTION
**Pull request by tomkerkhove**
Introduce PR welcome bot to guide new contributors

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/issues/4640

### CODE REVIEW

**Comment by tomkerkhove** [comment]
> Thank you for your contribution! 🙏 We will review your PR as soon as possible.
> 
> While you are waiting, make sure to:
> 
> * Add your contribution to all applicable KEDA versions
> * GitHub checks are passing
> * Is the DCO check failing? Here is [how you can fix DCO issues](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md?rgh-link-date=2023-06-09T11%3A24%3A45Z#i-didnt-sign-my-commit-now-what)
> 
> Learn more about:
> 
> * How to become a [listed end-user](https://github.com/kedacore/keda-docs#become-a-listed-keda-user)
> * How to become a [listed commerical offering](https://github.com/kedacore/keda-docs#become-a-listed-keda-commercial-offering)
> * Our [contribution guide](https://github.com/kedacore/keda-docs/blob/main/CONTRIBUTING.md?rgh-link-date=2023-06-09T11%3A24%3A45Z)

Is this good for you, @zroubalik?

**Review by zroubalik — APPROVED** [review]
LGTM

**Comment by zroubalik** [comment]
> how to become" to KEDA core as well

@tomkerkhove maybe make it one line? So the comment is not so huge:
```
How to become a [listed end-user](https://github.com/kedacore/keda-docs#become-a-listed-keda-user) and a [listed commerical offering](https://github.com/kedacore/keda-docs#become-a-listed-keda-commercial-offering)
```

File: .github/workflows/pr-bot-welcome.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - name: 'Comment on PR'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: ${{ github.event.number }},
+              body: 'Thank you for your contribution! 🙏 We will review your PR as soon as possible.\n\n\nWhile you are waiting, make sure to:\n\n\n- Add your contribution to all applicable KEDA versions\n- GitHub checks are passing\n- Is the DCO check failing? Here is [how you can fix DCO issues](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what)\n\n\nLearn more about:\n- How to become a [listed end-user](https://github.com/kedacore/keda-docs#become-a-listed-keda-user)\n- How to become a [listed commerical offering](https://github.com/kedacore/keda-docs#become-a-listed-keda-commercial-offering)\n- Our [contribution guide](https://github.com/kedacore/keda-docs/blob/main/CONTRIBUTING.md)'
```
Comment (tomkerkhove) [inline .github/workflows/pr-bot-welcome.yml:27]:
```suggestion
              body: 'Thank you for your contribution! 🙏 We will review your PR as soon as possible.\n\n\nWhile you are waiting, make sure to:\n\n\n- Add your contribution to all applicable KEDA versions\n- GitHub checks are passing\n- Is the DCO check failing? Here is [how you can fix DCO issues](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what)\n\n\nLearn more about:\n- How to become a [listed end-user](https://github.com/kedacore/keda-docs#become-a-listed-keda-user) or a [listed commerical offering](https://github.com/kedacore/keda-docs#become-a-listed-keda-commercial-offering)\n- Our [contribution guide](https://github.com/kedacore/keda-docs/blob/main/CONTRIBUTING.md)'
```

File: .github/workflows/pr-bot-welcome.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: PR Welcome Bot
+
+on:
+  pull_request:
```
Comment (JorTurFer) [inline .github/workflows/pr-bot-welcome.yml:4]: I'm not sure if `pull_request` can request a token with these permissions from forks

Comment (tomkerkhove) [reply .github/workflows/pr-bot-welcome.yml:4]: Correct! I forgot to update this PR
```suggestion
  pull_request_target:
```

**Pull request merged**